app/src/main/java/org/sopt/and/presentation/ui/my/MyViewModel.kt

```diff
@@ -0,0 +1,15 @@
+package org.sopt.and.presentation.ui.my
+
+import androidx.lifecycle.ViewModel
+import dagger.hilt.android.lifecycle.HiltViewModel
+import org.sopt.and.data.datasource.local.WaveLocalDataSource
+import javax.inject.Inject
+
+@HiltViewModel
+class MyViewModel @Inject constructor(
+    private val localDataSource: WaveLocalDataSource
+) : ViewModel() {
+    fun getLocalUserMail(): String {
+        return localDataSource.userEmail
+    }
```

Comment:
```suggestion
    fun getLocalUserMail(): String = localDataSource.userEmail
```
이렇게 하면 조금 더 코틀린스럽게 적을 수 있습니다.

Reply:
감사도리 ㅎㅎ 수정했사요